### ISSUE TITLE
Send daily summary of reviews to add-on developers

### ISSUE THREAD

**Issue by caitmuenster** [mozilla/addons-server#8604]
Related to issue #6817. 

When add-on developers receive email notifications about new user reviews, they want to be able to read the text of the reviews in the notification email. We should be able to provide this to them. 

@jvillalobos proposed that this could be achieved by sending a daily summary of new reviews to add-on developers. I would also be interested in exploring the possibility of creating a longer time delay between review edit and notification send to see if we could capture review text that way.

**Comment by jvillalobos** [comment]
I'm more in favor of the daily summary because:
1) Less email in general.
2) Less email to devs in case of review spamming.
3) More likely to show the "final" version of the review.
4) Easier to implement (I assume).

**Comment by alexander255** [comment]
Hi! I'm apparently the reason why this issue was opened.
I just wanted to comment that I think that sending daily digests instead of the current behaviour would be absolutely OK for me and the idea of making review spam less annoying (never really had this through) is probably a good one.
At least for me however, it would be quite useful if there was a seperate digest sent for each add-on, so that I can have them dropped into different mail folders on a per-add-on basis. This makes it easier to keep track of stuff for each add-on seperately.

**Comment by jvillalobos** [comment]
Agree. Doing a multi-add-on digest could be confusing and bloated for some devs.

**Comment by eviljeff** [comment]
Is this a replacement for the current email-per-rating we have now?

**Comment by caitmuenster** [comment]
Yes, ideally this would replace the current email-per-rating.
